cmd/integration-cip-gbg-karta/main.go

```diff
@@ -106,7 +107,7 @@ func bcGreenspaceRecord(ctx context.Context, cb application.ContextBrokerClient,
 				return err
 			}
 
-			logger.Info().Msgf("updated soilmoisture pressure value for %s (%s)", lon, lat)
+			logger.Info().Msgf("updated soilmoisture pressure value for %f %f", lon, lat)
```

Comment:
What information does this log output really supply? Who wants to see the coordinates in the log and why?